todo_app/data/view_model.py

```diff
@@ -0,0 +1,40 @@
+from todo_app.data.item import Item
+
+
+class ViewModel:
+    def __init__(self, items: list[Item]):
+        self._items = items
+ 
+    @property
+    def items(self):
+        return self._items
+    
+    @property
+    def todo_items(self):
+        output = []
+
+        for item in self._items:
+            if item.status == "To-Do":
+                output.append(item)
+
+        return output
```

Comment:
Nice; clean and simple 😄 More for comparison than suggestion, I always like to compare this with the arguably less readable, but definitely more concise, list-comprehension approach below:
```
    output = [ item for item in self._items if item.status == "To-Do"]
    return output
```